packages/workflow/src/api.ts

```diff
@@ -11,3 +44,268 @@ export {
   type WorkflowReadableStreamOptions,
   type WorkflowRun,
 } from '@workflow/core/runtime';
+
+export type {
+  CancelWorkflowRunParams,
+  CreateEventParams,
+  CreateEventRequest,
+  CreateHookRequest,
+  CreateStepRequest,
+  CreateWorkflowRunRequest,
+  GetHookParams,
+  GetStepParams,
+  GetWorkflowRunParams,
+  Hook,
+  ListEventsByCorrelationIdParams,
+  ListEventsParams,
+  ListHooksParams,
+  ListWorkflowRunStepsParams,
+  ListWorkflowRunsParams,
+  MessageId,
+  PaginatedResponse,
+  PauseWorkflowRunParams,
+  QueuePayload,
+  QueuePrefix,
+  ResumeWorkflowRunParams,
+  Step,
+  UpdateStepRequest,
+  UpdateWorkflowRunRequest,
+  ValidQueueName,
+} from '@workflow/world';
+
+/**
+ * Creates a workflow run using the configured World implementation.
+ */
+export function createRun(
```

Comment:
We should encourage users to use `getWorld()` directly instead, which makes the API more discoverable (i.e. what actions are available on a run? Just type `world.runs.` to find the list) and allows us to skip the duplication in this file